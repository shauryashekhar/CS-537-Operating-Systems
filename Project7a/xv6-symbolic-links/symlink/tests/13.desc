Creating a symlink with an existing symlink name. Should display ERROR.
