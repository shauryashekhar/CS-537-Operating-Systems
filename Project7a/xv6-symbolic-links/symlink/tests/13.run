cd xv6; ../../tester/run-xv6-command.exp CPUS=1 Makefile.test test_13 | grep SYMLINK; cd ..
