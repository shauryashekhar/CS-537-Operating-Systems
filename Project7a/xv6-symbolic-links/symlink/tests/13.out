SYMLINK: Symlink return = 0
SYMLINK: Symlink return = -1. You tried creating a symlink with an existing symlink name!
